<?xml version="1.0"?>

<robot name="erp42" xmlns:xacro="http://www.ros.org/xacro">
  
    <!-- ************************ Base Link ******************************** -->
   <link name= "base_footprint"/>

   <joint name="base_joint" type="fixed">
     <parent link="base_footprint"/>
     <child link="base_link"/>
     <origin xyz="0.0 0.0 0.53" rpy="0 0 0"/>
   </joint>

   <link name = "base_link">
    <visual>
        <origin xyz="0 0 0.055" rpy="0 0 0" />  
        <geometry>
            <box size = "2.02 0.985 0.906"/>
        </geometry>
        <material name="purple"/>
    </visual>
    <collision>
      <geometry>
            <box size = "2.02 0.985 0.4364"/>
        </geometry>
    </collision>

  </link>
<!-- ########################################Lidar################################### -->
<link name="lidar">
	    <collision>
	    	<origin xyz="0 0 0.071" rpy="0 0 0"/>
	      	<geometry>
	    		<cylinder radius="0.1" length="0.142"/>
	      	</geometry>
	    </collision>

	    <visual>
	    	<origin xyz="0 0 0.071" rpy="0 0 0"/>
	      	<geometry>
	    		<cylinder radius="0.1" length="0.142"/>
	      	</geometry>
	      	<material name="crusta"/>
	    </visual>
	</link>

	<joint name="lidar_joint" type="fixed">
	    <parent link="base_link"/>
	    <child link="lidar"/>
	    <origin xyz="0 0 0.508" rpy="0 0 0"/>
	</joint>
</robot>
